samples/GalleryApp/GalleryApp.csproj

```diff
@@ -133,7 +148,6 @@
   </ItemGroup>
   <ItemGroup>
     <Content Include="Assets\GitHub-Mark-32px.png" />
```

Comment:
>GitHub-Mark-32px.png [](start = 29, length = 20)

Could you please check with Varsha if there are any restrictions with using the GitHub logo in the gallery app?

Reply:
Will do.